.github/workflows/test.yml

```diff
@@ -202,7 +208,7 @@ jobs:
       - uses: docker-practice/actions-setup-docker@master
 
       - name: Start SQL Server ${{matrix.database}}
-        run: docker-compose -f docker-compose.yml up -d mssql-${{matrix.database}}
+        run: DOCKER_BUILDKIT=1 docker -f docker.yml up -d mssql-${{matrix.database}}
```

Comment:
This does not seem to work correctly.

https://github.com/prisma/tiberius/runs/5834177826?check_suite_focus=true

Reply:
```suggestion
        run: DOCKER_BUILDKIT=1 docker-compose -f docker-compose.yml up -d mssql-${{matrix.database}}
```

maybe?

Reply:
damn...same happened on Linux before. I am using a newer Docker version locally and in this docker-compose has been replaced with a "compose" subcommand inside "docker" (because docker-compose was using Python). I will just change this back to the docker-compose command.